### ISSUE TITLE
Provisioning new VM with vagrant fails because there is no `vagrant` user

### ISSUE THREAD

**Issue by develpr** [apache/incubator-openwhisk#3465]
## Environment details:

*Vagrant install quick start method used*

Host: macOS High Sierra
Vagrant: 2.0.2
VirtualBox: 5.2.8

## Steps to reproduce the issue:

1.   clone `https://github.com/apache/incubator-openwhisk`
2.  cd `<root of cloned repo>/tools/vagrant`
3.  `./hello`

## Provide the expected results and outputs:

I expected the results outlined in the readme / quick start

## Provide the actual results and outputs:

From what I can tell, the `vagrant` user isn't created during provisioning. I googled a bit but it's not clear to me "why" - I actually had thought that the creation of the `vagrant` user was a default part of provisioning a new VM via Vagrant, but... here we are:

```
...SNIP SNIP SNIP...
default: ++ echo OPENWHISK_TMP_DIR=/home/vagrant/.openwhisk/ansible/tmp
    default: ++ echo PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/snap/bin:/home/vagrant/bin:/home/vagrant/openwhisk/tools/build
    default: ++ cd /home/vagrant/openwhisk
    default: ++ cd tools/ubuntu-setup
    default: ++ su vagrant -c 'source all.sh'
    default: No passwd entry for user 'vagrant'
```

If I `vagrant ssh` into the box, and use this command which I _think_ will list all users:

```
ubuntu@ubuntu-xenial:~$ cut -d: -f1 /etc/passwd

root
daemon
bin
sys
sync
games
man
lp
mail
news
uucp
proxy
www-data
backup
list
irc
gnats
nobody
systemd-timesync
systemd-network
systemd-resolve
systemd-bus-proxy
syslog
_apt
lxd
messagebus
uuidd
dnsmasq
sshd
pollinate
ubuntu
vboxadd
```

Note there is no `vagrant` user on the list.

**Comment by chetanmeh** [comment]
@develpr  Whats the version of `ubuntu/xenial64` on your setup. It seems there was an [issue][1] with earlier version where vagrant user was not provisioned. This was fixed with v20180112.0.0.

On my setup (ubuntu as base machine) with recent version it works as expected
```
$ vagrant box list | grep xenial
ubuntu/xenial64   (virtualbox, 20180309.0.0)
```


[1]: https://bugs.launchpad.net/cloud-images/+bug/1569237

**Comment by develpr** [comment]
Confirmed! That was indeed the issue. I for some reason thought that Vagrant would automatically force an upgrade of the base box, or that the base box version constraint was somehow part of the `Vagrantfile`.

For anybody who finds this in the future, 

`vagrant box remove ubuntu/xenial64`

and then `vagrant up` (or `./hello`) seems to have resolved this issue for me.
